### ISSUE TITLE
Message Context Menu in Connect is Not Accessible

### ISSUE THREAD

**Issue by nickytonline** [forem/forem#12250]
**Describe the bug**

Connect has a contextual menu on a message that lets you edit or delete a menu. It appears when you hover over a message and when you hover over the three dots icon of the menu, it provides options. It cannot be accessed via the keyboard.

![image](https://user-images.githubusercontent.com/833231/104381717-ea09cc80-54fa-11eb-8fa5-318f9a7bc5bf.png)

**To Reproduce**

1. Tab through Connect. You'll be able to tab to focusable elements on a message like the user avatar or a link to the user, but that's it.
2. The menu never appears because the element is hidden until a message is hovered over with a mouse.

**Expected behavior**

A user should be able to access the contextual message menu and it's contents via keyboard or mouse.

**Screenshots**

https://user-images.githubusercontent.com/833231/104380587-3b18c100-54f9-11eb-9e0d-062539ad4b03.mp4

**Desktop (please complete the following information):**

- OS, version: N/A
- Browser, version: N/A

**Smartphone (please complete the following information):**

- Device: N/A
- OS, version: N/A
- Browser, version: N/A

**Additional context**

**Comment by narender2031** [comment]
Can I take this issue?

**Comment by rhymes** [comment]
Hi @narender2031, this has been reserved internally, as we're working on a accessibility "audit" on some pages. Thank you and sorry for the mishap

**Comment by nickytonline** [comment]
We can potentially build off the work done in #12578, but we don't necessarily need to make a generic component used in both spots as it may not be the right abstraction.
